### PR TITLE
[fix] pulsar is running on windows mistake has been fixed

### PR DESCRIPTION
**Pull request by ubo-dev**
This PR adds doc for [#21750](https://github.com/apache/pulsar/issues/21750)

This PR fixes [#21750](https://github.com/apache/pulsar/issues/21750)

### CODE REVIEW

File: docs/deploy-bare-metal-multi-cluster.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Deploying a multi-cluster Pulsar instance on bare metal consists of the followin
 
 ## System requirement
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
+Currently, Pulsar is available for 64-bit **macOS**, **Linux** and Pulsar is usable through Docker in **Windows**. See [Running Pulsar In Docker](https://pulsar.apache.org/docs/next/getting-started-docker/) for deeper understanding. To use Pulsar, you need to install 64-bit JRE/JDK.
```
Comment (visortelle) [inline docs/deploy-bare-metal-multi-cluster.md:25]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](https://pulsar.apache.org/docs/next/getting-started-docker/) if you want to Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
```

**Comment by visortelle** [comment]
@ubo-dev Thank you for the PR. I think the versioned docs should be also updated for these versions.

https://pulsar.apache.org/contribute/document-contribution/#update-versioned-docs

Screenshot 2024-03-03 at 12 28 39 AM

**Comment by ubo-dev** [comment]
@visortelle I've edited all the versions, I don't know if its too much :D

File: versioned_docs/version-2.11.x/deploy-bare-metal-multi-cluster.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ A Pulsar instance consists of multiple Pulsar clusters working in unison. You ca
 
 ## System requirement
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
+Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```
Comment (visortelle) [inline versioned_docs/version-2.11.x/deploy-bare-metal-multi-cluster.md:28]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK. For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

File: versioned_docs/version-2.11.x/deploy-bare-metal.md [hunk 3]
```diff
@@ -23,8 +23,7 @@ Deploying a Pulsar cluster consists of the following steps:
 
 ### Requirements
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
+Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```
Comment (visortelle) [inline versioned_docs/version-2.11.x/deploy-bare-metal.md:26]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK. For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

File: versioned_docs/version-3.0.x/deploy-bare-metal-multi-cluster.md [hunk 4]
```diff
@@ -25,8 +25,7 @@ A Pulsar instance consists of multiple Pulsar clusters working in unison. You ca
 
 ## System requirement
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
+Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```
Comment (visortelle) [inline versioned_docs/version-3.0.x/deploy-bare-metal-multi-cluster.md:28]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK. For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

File: versioned_docs/version-3.0.x/deploy-bare-metal.md [hunk 5]
```diff
@@ -23,8 +23,7 @@ Deploying a Pulsar cluster consists of the following steps:
 
 ### Requirements
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
+Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```
Comment (visortelle) [inline versioned_docs/version-3.0.x/deploy-bare-metal.md:26]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK. For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

File: versioned_docs/version-3.1.x/deploy-bare-metal-multi-cluster.md [hunk 6]
```diff
@@ -25,8 +25,7 @@ A Pulsar instance consists of multiple Pulsar clusters working in unison. You ca
 
 ## System requirement
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
+Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```
Comment (visortelle) [inline versioned_docs/version-3.1.x/deploy-bare-metal-multi-cluster.md:28]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK. For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

File: versioned_docs/version-3.1.x/deploy-bare-metal.md [hunk 7]
```diff
@@ -23,8 +23,7 @@ Deploying a Pulsar cluster consists of the following steps:
 
 ### Requirements
 
-Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.
-For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
+Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK.For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```
Comment (visortelle) [inline versioned_docs/version-3.1.x/deploy-bare-metal.md:26]:
```suggestion
Currently, Pulsar is available for 64-bit **macOS** and **Linux**. See [Run Pulsar In Docker](getting-started-docker.md) if you want to run Pulsar on **Windows**. To use Pulsar, you need to install 64-bit JRE/JDK. For the runtime Java version, please refer to [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation) according to your target Pulsar version.
```

**Comment by visortelle** [comment]
> I've edited all the versions, I don't know if its too much :D

I'm trying to figure it out in this issue:
https://github.com/apache/pulsar/issues/22177

@ubo-dev Thank you for the contribution! 🙂

**Comment by jak78** [comment]
Thanks @ubo-dev!

**Comment by ubo-dev** [comment]
@jak78 You're welcome my friend, I've enjoyed it :)

**Pull request merged**